bitcoin/src/blockdata/witness.rs

```diff
@@ -523,6 +546,21 @@ impl From<Vec<&[u8]>> for Witness {
     fn from(vec: Vec<&[u8]>) -> Self { Witness::from_slice(&vec) }
 }
 
+/// Tried to use a legacy uncompressed pubkey in a segwit v0 transaction.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct UncompressedPubkeyError;
```

Comment:
It looks like this error is not actually used anywhere yet. Am I missing something?

Reply:
Ouch, you are correct, it was used in an old version. Thanks, will fix.